core/src/views/UserMenu.vue

```diff
@@ -0,0 +1,174 @@
+<!--
+  - @copyright 2023 Christopher Ng <chrng8@gmail.com>
+  -
+  - @author Christopher Ng <chrng8@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+-->
+
+<template>
+	<NcHeaderMenu id="settings"
+		class="user-menu"
+		:aria-label="t('core', 'Open user menu')">
```

Comment:
This was:
```suggestion
		:aria-label="t('core', 'Open settings menu')">
```

not sure which is better.

Reply:
Not sure either but thinking was that the menu contains more than just settings, @nimishavijay @jancborchardt what should we label this?

Reply:
If no reply from the design team, the original strings should be used.
Keep in mind there is also admin entries in here. `user menu` might also be misleading

Reply:
Agreed with @skjnldsv that it could be misleading, also we're trying not to use the wording "user" so "settings menu" is probably a better fit anyway

Reply:
For the admin, it is personal settings...